doc/source/index.rst

```diff
@@ -0,0 +1,73 @@
+.. PBxplore API documentation master file, created by
+   sphinx-quickstart on Tue Oct  6 15:44:18 2015.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to PBxplore's documentation!
+====================================
+
+**PBxplore is a suite of tools dedicated to Protein Block analysis.**
+
+Protein Blocks (PBs) are structural prototypes defined by
+`de Brevern <http://www.dsimb.inserm.fr/~debrevern/index.php>`_ *et al* in 2000 [#]_.
+The 3-dimensional local structure of a protein backbone can be modelized as an 1-dimensional
+sequence of PBs.
+In principle, any conformation of any amino acid could be represented
+by one of the sixteen available Protein Blocks.
+
+.. figure:: img/PBs.jpg
+    :align: center
+
+    Schematic representation of the sixteen protein blocks, labeled from *a* to *p*
+    (Creative commons CC-BY).
```

Comment:
My comment was unclear. Obviously my edit arrived too late. I am talking 
about the CC licence for the figure here. CC by-sa is not just one 
license, there are several versions and international port of it. A link 
would tell exactly what CC license we are talking about. Also, a CC 
license always should come with a link.

On 12-01-16 16:59, Hub wrote:
>
> In doc/source/index.rst 
> <https://github.com/pierrepo/PBxplore/pull/110#discussion_r49472744>:
>
> > +====================================
> > +
> > +**PBxplore is a suite of tools dedicated to Protein Block analysis.**
> > +
> > +Protein Blocks (PBs) are structural prototypes defined by
> > +`de Brevern <http://www.dsimb.inserm.fr/~debrevern/index.php>`_ *et al* in 2000 [#]_.
> > +The 3-dimensional local structure of a protein backbone can be modelized as an 1-dimensional
> > +sequence of PBs.
> > +In principle, any conformation of any amino acid could be represented
> > +by one of the sixteen available Protein Blocks.
> > +
> > +.. figure:: img/PBs.jpg
> > +    :align: center
> > +
> > +    Schematic representation of the sixteen protein blocks, labeled from *a* to *p*
> > +    (Creative commons CC-BY).
>
> I plan to add it at the end of file after "Contact & Support". For 
> now, it's MIT but I don't know if we settled on that :)
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/pierrepo/PBxplore/pull/110/files#r49472744>.
>